src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -345,8 +345,12 @@ export default {
 		 * WebImage handlers
 		 */
 		async selectSocialAvatar() {
+			// FIXME: there must be a variables for this..:
+			const appUrl = 'http://localhost:8099/apps/contacts/'
```

Comment:
Yes, you can use the generateUrl function, you can import it if it's not available yet
`import { generateUrl } from '@nextcloud/router'`
https://github.com/nextcloud/forms/blob/46791f1f5c251f6ff5e42a4ffcfe78d5f2ee6d59/src/Forms.vue#L125

Reply:
Ah, good! I thought it wouldn't work when it just returned "/" and I expected it to start with http... Bin in fact it works well, thank you